aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -112,6 +112,8 @@ TORCH_LIBRARY_IMPL(aten, Batched, m) {
   // Tensor wrapper, it only has one dispatch key (Batched) on it. The resolution
   // here is to just directly call the underlying implementation.
   m.impl("size.int", static_cast<int64_t (*)(const Tensor&, int64_t)>(native::size));
+  m.impl("_add_batch_dim", native::_add_batch_dim);
```

Comment:
Shouldn't these functions also work on non-batched tensors too?

Reply:
They do work on non-batched tensors. They're here because BatchedTensor needs a registration for them, otherwise, `_add_batch_dim`/`_remove_batch_dim` only work on regular Tensors